README.md

```diff
@@ -22,10 +19,28 @@ Proposal to define an XML-based logging format for outputs from EDA tools and lo
 * State Coverage
 * Transition Coverage
 
-## Functional Coverage
+### Functional Coverage
 
 * Coverage Model
   * Dimension
   * Bin
     * Item
     * Range
+
+### Synthesis
+
+* Area/resources
+* Timing
+
+### Implementation
+
+* Area/resources
+* Timing
+
+# References
+
```

Comment:
structured output format from Xilinx Vivado

Reply:
The summary now explains that multiple log/report formats are expected to be readed. Let's not specify which formats are used for Vivado, until we can double check the features in the latest versions.